<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>569</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>954</width>
    <height>569</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>954</width>
    <height>569</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TuringSimulator</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>611</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>GraphicView</string>
     </attribute>
     <widget class="QToolButton" name="StateButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add State</string>
      </property>
      <property name="text">
       <string extracomment="Etat">Etat</string>
      </property>
      <property name="icon">
       <iconset resource="../Ressource.qrc">
        <normaloff>:/new/icones/Etat.png</normaloff>:/new/icones/Etat.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="AcceptButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add Initial State</string>
      </property>
      <property name="text">
       <string extracomment="Etat Final">Etat final</string>
      </property>
      <property name="icon">
       <iconset resource="../Ressource.qrc">
        <normaloff>:/new/icones/Accept.gif</normaloff>:/new/icones/Accept.gif</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="EditButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Edit</string>
      </property>
      <property name="text">
       <string extracomment="Editer">Editer</string>
      </property>
      <property name="icon">
       <iconset resource="../Ressource.qrc">
        <normaloff>:/new/icones/Edition.png</normaloff>:/new/icones/Edition.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="TransitionButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add Transition</string>
      </property>
      <property name="text">
       <string extracomment="Transition">Transition</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="RejectButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add Final State</string>
      </property>
      <property name="text">
       <string extracomment="Etat Final">Etat final</string>
      </property>
      <property name="icon">
       <iconset resource="../Ressource.qrc">
        <normaloff>:/new/icones/interdit.jpg</normaloff>:/new/icones/interdit.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tapes:</string>
      </property>
     </widget>
     <widget class="QToolButton" name="DeleteButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Delete Item</string>
      </property>
      <property name="text">
       <string extracomment="Transition">Delete</string>
      </property>
      <property name="icon">
       <iconset resource="../Ressource.qrc">
        <normaloff>:/new/icones/delete.png</normaloff>:/new/icones/delete.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="EditTransitionButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Edit Tape</string>
      </property>
      <property name="text">
       <string extracomment="Transition">Edit</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="AddTapeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add Tape</string>
      </property>
      <property name="text">
       <string extracomment="Transition">Add</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="DeleteTapeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Remove Tape</string>
      </property>
      <property name="text">
       <string extracomment="Transition">Remove</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>TextView</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>581</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="bouton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Simulation</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="console">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>311</width>
      <height>431</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Console</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTuringSim">
    <property name="title">
     <string>TuringSimulator</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionFermer"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEditer">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopier"/>
    <addaction name="actionColler"/>
    <addaction name="actionCouper"/>
   </widget>
   <addaction name="menuTuringSim"/>
   <addaction name="menuEditer"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveau">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCouper">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Ressource.qrc"/>
 </resources>
 <connections/>
</ui>
